src/ServerDaemon.chpl

```diff
@@ -774,6 +773,7 @@ module ServerDaemon {
             this.socket = this.context.socket(ZMQ.REP); 
             this.port = try! getEnv('METRICS_SERVER_PORT','5556'):int;
 
+            try! this.socket.bind("tcp://*:%?".format(this.port));
             try! this.socket.bind("tcp://*:%?".format(this.port));
```

Comment:
I think this is the second time this has snuck in lol

Reply:
Thanks for catching this! (merge conflicts have not been my favorite lately)